### ISSUE TITLE
package jove

### ISSUE THREAD

**Issue by peterjc** [conda-forge/staged-recipes#10904]
As described nicely in the following article, jove (Jonathan's own version of Emacs) is a lightweight alternative to emacs:

https://opensource.com/article/17/1/jove-lightweight-alternative-vim

Debian and Fedora packages:

https://packages.debian.org/search?keywords=jove
https://apps.fedoraproject.org/packages/jove

Upstream:

ftp://ftp.cs.toronto.edu/pub/hugh/jove-dev/

Current latest release is Jove v4.16.0.73 from July 2010,

ftp://ftp.cs.toronto.edu/pub/hugh/jove-dev/jove4.16.0.73.tgz

There is an experimental release v4.17.0.74 from October 2015,

ftp://ftp.cs.toronto.edu/pub/hugh/jove-dev/experimental/jove4.16.0.74.tgz

**Comment by peterjc** [comment]
Quoting ftp://ftp.cs.toronto.edu/pub/hugh/jove-dev/README here:

> ftp.cs.utoronto.ca:/pub/hugh/jove-dev/ is the site for development
> versions of JOVE, a text editor.
> 
> Released versions of JOVE appear in ftp.cs.utoronto.ca/pub/moraes/jove/
> 
> Many files here are named with only version numbers (plus a suffix
> indicating compression technique).  They are tar files containing the
> development version of JOVE with that version number.  Note that the
> tar file unpacks files into ".".
> 
> 4.16.0.38.MSWind32.exe is a self-extracting ZIP file that contains
> JOVE compiled for MS WIN32 (MS Windows 95, 98, and NT).  If you don't
> trust random .exe files, you can unzip this file.  But, of course, one
> of the unpacked files is a random executable -- jove.exe!!
> 
> 4.16.0.41.jovedoss.zip is source suitable for compiling under MSDOS
> 4.16.0.41.jovedosx.zip is compiled for MSDOS
> 
> jove-4.16.0.44-1.src.rpm is a source RPM (for LINUX).
> jove-4.16.0.44-1.i386.rpm is a binary RPM for a RedHat 5.2+updates system.
> 
> Any file ending in .sig is a PGP signature for a file with the
> corresponding name.  The public key is of hugh@mimosa.com.  Get that
> key through some other channel (http://www.uk.pgp.net/pgpnet/wwwkeys.html
> has a copy).
> 
> Hugh Redelmeier  hugh@mimosa.com  +1 416 482-8253
> 

Note is says the latest release is here:

http://ftp.cs.utoronto.ca/pub/moraes/jove/

That currently is Jove v4.17.0.5 from Jan 2020,

http://ftp.cs.utoronto.ca/pub/moraes/jove/jove-4.17.0.5.tgz

The README at http://ftp.cs.utoronto.ca/pub/moraes/jove/jove.README says:

> TL;DR See github.com/jonmacs/jove
> 
> For a long time, this anonymous ftp directory was the
> official distribution of the Jove text editor (Jonathan's
> Own Version of Emacs).
> 
> No official releases have happened since version 4.16 (in
> 1996), which is now in ARCHIVE/4.16 for historical purposes
> (I hate deleting anything).
> 
> Over the period 1997-2010, D. Hugh Redelmeier put test
> releases 4.16.0.24 through 4.16.0.73 in /pub/hugh/jove-dev,
> accumulating improvements and bugfixes from many of the Jove
> developers.  They never became an official 4.17 tarball, but
> they did make their way into Debian and some other Linux distros.
> 
> /pub/hugh/jove-dev/experimental/jove4.16.0.74.tgz contains a
> 2015 batch of fixes, which is arguably the most recent 
> "release" of Jove (though I hope that'll change one day and
> I'll get to modify this note!)
> 
> My personal archive of Jove history is on
> github.com/jonmacs/jove/, where the master branch has all
> historical commits I had, from Jonathan Payne's 4.3 (the
> version that was distributed and documented with BSD)
> through to 4.16.0.74.
> 
> My own changes will accumulate in the moraes branch of that repo,
> and might get promoted to master eventually, as 4.17.
> 
> 	Mark Moraes, Jan 22, 2020.
>

Looking at the new github repository https://github.com/jonmacs/jove/ there is an even more recent tagged release:

https://github.com/jonmacs/jove/tree/4.17.3.6

**Comment by peterjc** [comment]
Cross reference https://github.com/jonmacs/jove/issues/2
